### ISSUE TITLE
How would you use Tensorboard to keep track of training in tf-object-detection-sagemaker

### ISSUE THREAD

**Issue by MishallSwain** [isobar-us/multilabel-image-classification-tensorflow#7]
I would like to use Tensorboard to monitor training loss and accuracy. I have been using TensorFlow API to train model directly, but I'm not sure how this would happen through AWS Sagemaker training. 

Also, how would you go about continuing training if the model isn't converged in the defined number of steps? Like is saving the checkpoints and continuing training from there possible?

I have just started using AWS Sagemaker, so excuse me if this is a silly question.
